.gitignore

```diff
@@ -1,5 +1,6 @@
 __pycache__
 .cache
+.git
```

Comment:
I don't think you need to tell git to ignore itself. Did you mean to put this in `.dockerignore`?

Reply:
Er, brain fart; yeah.  Will fix.